packages/lit-dev-tools-esm/src/dev-server.ts

```diff
@@ -72,6 +73,9 @@ const removeWatchScriptFromPlaygroundFiles: DevServerPlugin = {
   },
 };
 
+const clientId = 'FAKE_CLIENT_ID';
+const clientSecret = 'FAKE_APP_SECRET';
```

Comment:
Could we add a comment here as well that this should never be replaced with the real secret.